samples/non-native/pom.xml

```diff
@@ -0,0 +1,167 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can't you use existing sample `java-application` which already doesn't have tests? See: https://github.com/graalvm/native-build-tools/tree/master/samples/java-application.

Reply:
But is that real scenario then? Why would someone have classes in `src/test` that doesn't have tests? I am okay with the change, but I am just wondering if it has real usecase?